frontend/src/components/pages/RegistrantExperience/RegistrationSteps/AdditionalInfo/QuestionCards/EDLPSelectionControl.tsx

```diff
@@ -0,0 +1,71 @@
+import {
+  FormControl,
+  FormErrorMessage,
+  FormLabel,
+  Radio,
+  RadioGroup,
+  Text,
+  VStack,
+} from "@chakra-ui/react";
+import React from "react";
+import RequiredAsterisk from "../../../../../common/RequiredAsterisk";
+
+type EDLPSelectionControlProps = {
+  requireEDLP: boolean | null;
+  setRequireEDLP: (setRequireEDLP: boolean) => void;
+  nextClicked: boolean;
+};
+
+const EDLPSelectionControl = ({
+  requireEDLP,
+  setRequireEDLP,
+  nextClicked,
+}: EDLPSelectionControlProps): React.ReactElement => {
```

Comment:
is there a reason we chose to type this as React.ReactElement? It's not incorrect it's just that i think the majority of our code base has it as JSX.Element so consistency may be prefered? Not strongly opinionated on this but it's a consideration in my mind